feat: prebidAdID and consumption on android

brianreidy opened the pull request:
tags: prebidAdID, setPropPrebidAdID, setPrebidAdID

tickets: [PM-480](https://patchdev.atlassian.net/browse/PM-480)

brianreidy committed:
feat: prebidAdID and consumption on android

tags: prebidAdID, setPropPrebidAdID, setPrebidAdID

tickets: [PM-480](https://patchdev.atlassian.net/browse/PM-480)

cogell commented on RNPublisherBanner.js:
```diff
@@ -98,6 +99,11 @@ PublisherBanner.propTypes = {
   adUnitID: string,
 
+  /**
+   * DFP ad unit ID
```
this comment correct?

brianreidy replied:
These are not the comments you are looking for. fixed

brianreidy committed:
fix: add prebid android sdk

tags: sdk, android, prebid

tickets: [PM-480](https://patchdev.atlassian.net/browse/PM-480)
